src/utils/BaseUtil.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+
+namespace SwitchViews\utils;
```

Comment:
It is ideal to that namespace names should start with Capital Letter. In this case **utils** should be **Utils**